src/data-root.ts

```diff
@@ -52,11 +67,12 @@ export async function update(
   // Make API call
   await Promise.all([
     ipfs.reconnect(),
-    fetch(`${apiEndpoint}/user/data/${cid}`, {
+    fetchWithRetry(`${apiEndpoint}/user/data/${cid}`, {
       method: 'PATCH',
-      headers: {
-        'authorization': `Bearer ${jwt}`
-      }
+      headers: { 'authorization': `Bearer ${jwt}` },
+      retries: 100,
+      retryOn: [ 503, 504 ],
```

Comment:
503s as well?

Reply:
Yeah, I imagine we could get a 503 "Service Unavailable" if IPFS is currently unavailable (eg. ipfs server is restarting). Is that a thing?

Reply:
Ah perhaps (honestly not sure how our server deals with it). Seems good either way tho :ok_hand: